README.md

```diff
@@ -55,7 +55,9 @@ Linux or OS X.)
 Then, in another terminal session run this (**an error about not
 finding `github.com/PursuanceProject/pursuance` is OK here**) --
 
-``` $ go get ./... ```
+``` $ go get ./.. ```
```

Comment:
@wittnl This still needs the full 3 `.`s, but good point about `npm run build` :+1:

Reply:
Ah it's a go thing, I'll revert that.